src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -46,20 +46,26 @@
     :render-fn                         render-fn}])
 
 (defn segments-community-lists [selected-tab]
-  (let [communities (rf/sub [:communities/community-ids])
+  (let [ids-by-user-involvement (rf/sub [:communities/community-ids-by-user-involvement])
         tab @selected-tab]
     [rn/view {:style {:padding-left     20
                       :padding-right    8
                       :padding-vertical 12}}
      (case tab
        :joined
-       [communities-list communities]
+       [communities-list (:joined ids-by-user-involvement)]
 
        :pending
-       [communities-list communities]
+       [communities-list (:pending ids-by-user-involvement)]
 
        :opened
-       [communities-list communities])]))
+       [communities-list (:opened ids-by-user-involvement)]
+
+       :default
```

Comment:
Just a minor correction here @vkjr, the default branch in the `case` macro doesn't take an option, only the `cond` macro understands that.

```clojure
;; This explodes :boom: 
(let [tab :unknown]
  (case tab
    :joined  :joined
    :pending :pending
    :opened  :opened
    :error))

(let [tab :unknown]
  (cond 
    (= tab :joined)  :joined
    (= tab :pending) :pending
    (= tab :opened)  :opened
    :default         :error))
```

Reply:
🤦 , thanks for noticing!